### DISCUSSION TITLE
My use case: use org mode and ekg at the same time

### DISCUSSION THREAD

**Discussion by webloginwu** [ahyatt/ekg#141] [Ideas]
**I like some features of ekg, for example**

1. embeding
2. display notes related to multiple tags 

**My use case is:**

1.I already invest a lot in org mode
2.I want to export data in org mode to ekg
4.use features provided by ekg to meet my requirement
5.continue to use org mode
6.clean database, export all data in org mode to ekg again
7.use features provided by ekg to meet another requirement
8.Mybe I will switch to ekg eventually if it can fully meet my requirement

**My concern**
if data in database is lost, I lost every thing.
I need backup solution, currently I prefer to backup data in org mode, because it is readable even emacs is not availavle.

**one feature I wanted**
 I'd like to show one note with the tag that 100% match the content of heading in org mode, currently it seems ekg use fuzzy match

**Comment by ahyatt** [comment]
Thanks for the note! (and hey what happened to point 3).  I agree that what you want is reasonable.  The solution we have is to export to Logseq, which will export to org mode documents.  It may not export to a format you like, but the nature of ekg is that it is taking advantage of things that are awkward to do in files.  So a note-to-file export is possible but you'd end up with a lot of files with opaque names, which to me doesn't seem that useful.  If you disagree, I'd love to hear it, because it's fairly easy to write such an exporter.

Can you expand a bit on the feature you mention?  I have a branch (not currently 100% working) that does do exact searches on note text and tags, so maybe that's what you want?  But fetching by tags is exact right now.

**Comment by jayrajput** [comment]
This looks like `ekg-denote` package which I have been working on forever :-), but cannot spend time. The package allows you to export ekg notes to denote and import from denote to ekg. The package will have capability where you can switch between ekg note and denote files.

**Comment by webloginwu** [comment]
point 3:
I like text based orientation, it is easy to share and backup
but I foud weakness point in org mode
I have 9 .dir-locals.el files for org-roam-db-location for 9 projects
every time I switch project, or update files in project
In order to make org link work, I have to build database (M-x org-roam-db-sync) frequently, it annoys me a lot under high pressure in compay.
this is why I try to test ekg to see if there are some use case I can use ekg, and in the meanwhile I want to keep using org-roam
